hack/e2e-run-notebook.sh

```diff
@@ -42,7 +42,7 @@ print_results() {
     kubectl describe trainjob
     kubectl logs -n kubeflow-system -l app.kubernetes.io/name=trainer
     kubectl logs -l jobset.sigs.k8s.io/replicatedjob-name=trainer-node,batch.kubernetes.io/job-completion-index=0 --tail -1
-    kubectl wait trainjob --for=condition=Complete --all --timeout 3s
+    kubectl wait trainjob --for=condition=Complete --all --timeout 600s
```

Comment:
Why do we want to increase timeout here ?
We expect that when Jupyter Notebook is finished, TrainJob should be in the Complete status.

Reply:
ok